libraries/procedural/src/procedural/Procedural.h

```diff
@@ -40,7 +40,7 @@ const size_t MAX_PROCEDURAL_TEXTURE_CHANNELS{ 4 };
  *     If a procedural material contains a vertex shader, the bounding box of the material entity is used to cull the object to which the material is applied.
  * @property {string} fragmentShaderURL - A link to a fragment shader.  Currently, only GLSL shaders are supported.  The shader must implement a different method depending on the version.
  *     <code>shaderUrl</code> is an alias.
- * @property {string[]} channels=[] - An array of input texture URLs.  Currently, up to 4 are supported.
+ * @property {string[]} channels=[] - An array of input texture URLs or entity IDs.  Currently, up to 4 are supported.
```

Comment:
```suggestion
 * @property {string[]} channels=[] - An array of input texture URLs or Material entity IDs.  Currently, up to 4 are supported.
```
I assume?

Reply:
image or web entities.  generally, anything that supported texture URLs can now reference those types by ID, and just material entities can refer to material entities.  I’m not sure how to best describe this in the docs succinctly, since they’re referenced all over the place

Reply:
I've updated my suggested change. The API docs should say what acceptable values are / how a property can be used - even if this information is repeated multiple times at different locations because a user typically will be just looking at one particular piece of the docs, not reading the whole thing.